src/modules/Cache.ts

```diff
@@ -53,4 +59,25 @@ export class Cache {
 	remove(url: string): void {
 		this.pages.delete(this.getCacheUrl(url));
 	}
+
+	/**
+	 * Delete all expired pages
+	 */
+	deleteExpiredPages(): void {
+		for (const [key, page] of this.pages.entries()) {
+			if (this.isExpiredPage(page)) {
+				this.pages.delete(key);
+			}
+		}
+	}
+
+	/**
+	 * Checks if a page should be considered expired.
+	 * Will return true if the page was saved more then 10 minutes ago
+	 */
+	isExpiredPage(page: PageRecord): boolean {
+		const ttl = this.swup.options.getTimeToLive(page);
+		if (ttl === 0) return false;
```

Comment:
Making `0` never delete the cached pages seems counterintuitive. I would expect such a value to always delete the cache (basically act as `cache: false`).